install_config/install/docker_registry.adoc

```diff
@@ -20,6 +20,18 @@ link:../../architecture/infrastructure_components/image_registry.html#integrated
 Docker registry] that can be deployed in your OpenShift environment to locally
 manage images.
 
+To make full use of the registry, you must create a link:../../architecture/core_concepts/projects_and_users.html#users[regular user] with the `system:registry` role, if one does not already exist, using your preferred link:../../admin_guide/configuring_authentication.html[identity provider].
```

Comment:
There's some inconsistent markup between roles and users mentioned in the topic. E.g., compare system:registry,  system:image-builder, system:admin, etc. They should probably be the same; the precedent was using plain bold as ["system items"](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference) (fuzzy, I know).